torch_xla/torch_xla.py

```diff
@@ -215,6 +219,9 @@ def _compile():
       torch_xla._XLAC._set_use_eager_mode(saved_eager_mode_status)
       torch_xla._XLAC._set_current_graph_name(saved_current_graph_name)
 
+  custom_compile_options = custom_compile_options or {}
+  if len(custom_compile_options) > 0:
```

Comment:
This means there's no way for the user to clear the custom compile options?

Reply:
The behavior of compile() should be consistent with set_custom_compile_options(). If given an empty dict, both should do the same thing. Otherwise the API is very confusing and error-prone.

I suggest:

- keep set_custom_compile_options() unchanged,
- treat None as no-op here,
- treat {} as "clear options" here.